sushichef.py

```diff
@@ -0,0 +1,245 @@
+
+    video_data = []
+    pp = pprint.PrettyPrinter()
+    try:
+        video = ArvindVideo(
+                    url=src, 
+                    title=title, 
+                    description=description, 
+                    lang_code=lang_code,
+                    filename_prefix=filename_prefix)
+        video_data.append(video)
+
+    except Exception as e:
+        print('==> Error scraping video URL', url)
+        pp.pprint(e)
+    return video_data
+
+def download_videos(lang_code):
+    """
+    Scrape, collect, and download the videos and their thumbnails.
+    """
+    video_data = []
+    vinfo = DATA[lang_code]['video_info']
+    try:
+        download_path = vinfo['download_path']
+        video_data = scrape_video_data(
+                            vinfo['video_url'], 
+                            lang_code, 
+                            vinfo['filename_prefix'])
+        print('==> DOWNLOADING', download_path, '====> vinfo', vinfo)
+        # Download video and metadata info for all video objects collected.
+        for i, video in enumerate(video_data):
+            progress = '%d/%d' % (i+1, len(video_data),)
+            progress = '==> %s: Downloading video from %s ...' % (progress, video.url,)
+            print(progress)
+            if video.download(download_dir=download_path, video_data=DATA):
+                # TODO(cpauya): Create VideoTopic then add to channel.
+                pass
+    except Exception as e:
+        print('Error downloading videos:')
+        pp = pprint.PrettyPrinter()
+        pp.pprint(e)
+        raise e
+    print('==> DONE downloading videos for', vinfo)
+    return video_data
+
+def build_english_video_topics(topic):
+    """
+    """
+    video_data = download_videos(LANG_CODE_EN)
+    if not video_data:
+        print('==> Download of Videos FAILED!')
+        return False
+
+
+    for i, video in enumerate(video_data):
+        filepath = video.filepath
+        video_node = VideoNode(
+                source_id=video.uid, 
+                title=video.title, 
+                description=video.description,
+                aggregator=LE,
+                thumbnail=video.thumbnail,
+                license=get_license("CC BY-NC-SA", copyright_holder=POINTB),
+                files=[
+                    VideoFile(
+                        path=filepath,
+                        language=LANG_CODE_EN
+                    )
+                ])
+        topic.add_child(video_node)
+
+    return topic
+
+
+class ArvindChef(SushiChef):
+    channel_info = {
+        "CHANNEL_TITLE": "Arvind Gupta Toys",
+        # where you got the content (change me!!)
+        "CHANNEL_SOURCE_DOMAIN": "arvindguptatoys.com",
+        # channel's unique id (change me!!) # NOTE when you remove test- the channel_id will change; make sure to update notion card
+        "CHANNEL_SOURCE_ID": "beta-arvind",
+        "CHANNEL_LANGUAGE": "en",  # le_utils language code
```

Comment:
Please change this to `mul` for multiple languages, see
https://github.com/learningequality/le-utils/blob/master/le_utils/resources/languagelookup.json#L6-L9